db/migrate/20160623133301_change_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnName < ActiveRecord::Migration
+  def change
+    rename_column :dishes, :allergy_info, :allergy_info
```

Comment:
Since we are creating a new object, I'm worried the test is not actually routing the user to the right show page.